rest/resource_path_test.go

```diff
@@ -3,8 +3,8 @@ package rest
 import (
 	"testing"
 
-	mem "github.com/rs/rest-layer/resource/testing/mem"
 	"github.com/rs/rest-layer/resource"
+	mem "github.com/rs/rest-layer/resource/testing/mem"
```

Comment:
Could remove `mem ` prefix for consistency - not needed, and no longer different from the last bit of the import path

Reply:
@smyrman You're right... but for me right now that's an "out of scope" decision for this PR because it's not a strict change by `gofmt`. Please feel welcome to make modifications however you want, though.